.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -1,5 +1,6 @@
 <component name="InspectionProjectProfileManager">
   <settings>
+    <option name="PROJECT_PROFILE" value="Default" />
```

Comment:
you can use [.gitignore](https://git-scm.com/docs/gitignore) file to make sure that such files will not get added into your repository

Reply:
especially the binary files from the `__pychache__`